keyboards/binepad/bnk9/config.h

```diff
@@ -0,0 +1,22 @@
+// Copyright 2023 binepad (@binepad)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#define ENCODER_DEFAULT_POS 0x3  // enable 1:1 resolution
+
+#ifdef RGB_MATRIX_ENABLE
+    #define WS2812_PIO_USE_PIO1
+    // #define RGB_MATRIX_TIMEOUT 600000  // 10 minutes
+    #define RGB_DISABLE_WITH_FADE_OUTS
+    #define RGB_DISABLE_WHEN_USB_SUSPENDED
+    #define RGB_MATRIX_LED_PROCESS_LIMIT 9  // All nine keys
+    #define RGB_MATRIX_MAXIMUM_BRIGHTNESS 180  // Not too bright so that the LED's don't burn out
+
+    // Timing for SK6812
+    #define WS2812_TIMING 1250
+    #define WS2812_T0H    350
+    #define WS2812_T0L	  (WS2812_TIMING - WS2812_T0H)
+    #define WS2812_T1H	  650
+    #define WS2812_T1L	  (WS2812_TIMING - WS2812_T1H)
+#endif
```

Comment:
- Not using another PIO based driver
- `RGB_DISABLE_WITH_FADE_OUTS` is not a thing
- Duplication of default values
- Items should be migrated to info.json

```suggestion
// Timing for SK6812
#define WS2812_T1H	  650
```

Reply:
On the PIO item - if I use the default `PIO0` on this board then I get flickering white flashes on the RGB.  *(Don't know why ... but* `PIO1` *seems to work)*

Reply:
Then you at least need to add a comment, otherwise the line it just going to get removed at a later date for the `Not using another PIO based driver` reasoning.

Reply:
Done